src/Rules/Install.hs

```diff
@@ -193,6 +193,20 @@ installPackages = do
                 let ghcCabalInplace = inplaceBinPath -/- "ghc-cabal" -- HACK?
                 need [ ghcCabalInplace ]
 
+                let cabalFile = pkgCabalFile pkg
+                -- HsColour sources
+                -- QUESTION: what is the output of GhcCabalHsColour?
+                whenM (isSpecified HsColour) $ do
+                    top <- interpretInContext context getTopDirectory
+                    let installDistDir = top -/- buildPath context
+                    -- HACK: copy stuff back to the place favored by ghc-cabal
+                    quietly $ copyDirectoryContents (Not excluded)
```

Comment:
Wait, I'm confused now.

We need to run `build $ Target context GhcCabalHsColour [cabalFile] []` during normal builds. Why are you moving this to the install rule? 

I would expect to see `GhcCabalHsColour` to be left in the documentation rule, while the install rule should simply copy all resulting files. Isn't this sufficient?